generate/main.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+)
+
+// ServerVersion : the version from the upstream repository
+var ServerVersion = getDefaultVersion()
+
+// GrafeasRepository : the repository of the Grafeas server to use for downloading Swagger files
+var GrafeasRepository = "https://github.com/grafeas/grafeas"
+
+// Reference : the commit, branch, or tag to use for downloading Swagger files from the defined Grafeas repo
+var Reference = "master"
+
+// APIVersion : the version of the Grafeas API per the Google Cloud scheme - https://github.com/grafeas/grafeas/blob/master/docs/versioning.md#api
+var APIVersion = "v1beta1"
+
+// SwaggerCodegenVersion : the version of the Swagger CodeGen CLI to download
+var SwaggerCodegenVersion = "2.4.5"
+
+// MergedClient : whether to keep the generated Swagger clients separate or to merge the paths
+// TODO: implement function that merges the paths of each Swagger spec into one file before running codegen
+var MergedClient = false
+
+// trimVersionTag : remove the "v" prefix from the version tag string
+func trimVersionTag(tag string) string {
+	if strings.HasPrefix(tag, "v") {
+		return strings.TrimPrefix(tag, "v")
+	}
+	return tag
+}
+
+func getDefaultVersion() string {
+	version := os.Getenv("VERSION")
+	if version == "" {
+		tag := getMostRecentTag()
+		trimmedVersion := trimVersionTag(tag)
+		version = trimmedVersion
+	}
+	return version
+}
+
+func getMostRecentTag() string {
+	tagsBytes, _ := get("https://api.github.com/repos/grafeas/grafeas/tags")
+	var tags []map[string]interface{}
+	json.Unmarshal(tagsBytes, &tags)
+	return tags[0]["name"].(string)
+}
+
+func swaggerCompatibility(jsonInput string) string {
+	re := regexp.MustCompile(`\/{.*(=.*)}`)
```

Comment:
Small comment on this: because `GetNote: "/v1beta1/{name=projects/*/notes/*}"` and `GetOccurrence: "/v1beta1/{name=projects/*/occurrences/*}"` both share the same form of /v1beta1/{name}, the swagger generation only renders GetOccurrence.
I don't think there's a way of fixing this other than changing the grafeas.swagger.json.